tool/typedoc-theme.js

```diff
@@ -53,16 +68,16 @@ class SassSiteRenderContext extends DefaultThemeRenderContext {
         // The first line is arguments to impl_status, anything after that is the
         // contents of the block.
         const lineBreak = text.indexOf('\n');
-        const firstLine =
-          lineBreak === -1 ? text : text.substring(0, lineBreak);
+        const firstLine = parseCompatibility(
+          lineBreak === -1 ? text : text.substring(0, lineBreak),
+        );
         const rest =
           lineBreak === -1 ? null : text.substring(lineBreak + 1).trim();
-
         return JSX.createElement(JSX.Raw, {
           html:
-            `<% impl_status(${firstLine}) ${rest ? 'do' : ''} %>` +
-            context.markdown(rest) +
-            (rest ? '<% end %>' : ''),
+            `{% compatibility ${firstLine} %}` +
+            (rest ? context.markdown(rest) : '') +
```

Comment:
This is an existing bug on the production site, but it looks like TypeDoc stops parsing this text as soon as it hits HTML, e.g. a ` or link or something. See e.g. expanded compatibility table at: https://sass-lang.com/documentation/js-api/interfaces/LegacySharedOptions#includePaths

Reply:
Tracking this down a bit more, it looks like `compat.content[0].text` is the issue. In cases with embedded HTML, `compat.content` looks like:

```
[
  {
    kind: 'text',
    text: 'dart: true, node: "3.0.0"\n' +
      '\n' +
      'Versions of Node Sass before 3.0.0 don’t support arrays of importers, nor\n' +
      'do they support importers that return '
  },
  { kind: 'code', text: '`Error`' },
  {
    kind: 'text',
    text: ' objects.\n\nVersions of Node Sass before 2.0.0 don’t support the '
  },
  { kind: 'code', text: '`importer`' },
  { kind: 'text', text: ' option at\nall.' }
]
```

Reply:
https://github.com/sass/sass-site/issues/730